### ISSUE TITLE
Chapter 15, page 506, wrong image

### ISSUE THREAD

**Issue by gabramson** [rasbt/python-machine-learning-book-2nd-edition#60]
The image on page 506 doesn't seem to match the text. The image is supposed to display pooling, but instead it shows convolution.

![image](https://user-images.githubusercontent.com/31111688/35631290-1545fd3a-0669-11e8-83ec-12ddbfa34d77.png)

Best regards,
Gary

**Comment by mirjalil** [comment]
Yes, my sincere apology! Unfortunately during the final round of edits, this figure was misplaced with a duplicated figure from convolution section. The correct figure corresponding to the pooling is the following:

![15_07](https://user-images.githubusercontent.com/2880929/35631838-faa1aa1a-0672-11e8-9d24-2f825137000c.png)

**Comment by gabramson** [comment]
No problem. Thanks for the quick response!

**Comment by rasbt** [comment]
Yeah, this is really unfortunated, not sure what happened during the final layout stage, since it looked fine when Vahid & I were reviewing the layouted drafts. Sorry about that. At least, the figures should be all in the correct order in the Jupyter notebooks though.
